src/backend/utils/adt/graphid.c

```diff
@@ -277,3 +280,36 @@ Datum graphid_hash_cmp(PG_FUNCTION_ARGS)
 
     PG_RETURN_INT32(hash);
 }
+
+int64 get_new_id(char entity_type, Oid graphoid) {
+    Oid seq_id;
+    int count;
+
+    // get the OID of sequence
+    if (entity_type == LABEL_KIND_VERTEX) {
```

Comment:
We keep curly brackets in a separate line. https://age.apache.org/contribution/guide/

Reply:
Also, there are extra spaces at the end of lines that needs to be removed.